### PR TITLE
fix(ticket #202) fix hook

### PR DESCRIPTION
**Pull request by muhammad-bilal-ali-khokhar**
### Description
- This PR addressing a bug fix in `useGetLoggedInUser.ts` hook , this hook want returning the promise , now i have fixed this issue and getting correct data in return

### CODE REVIEW

File: src/hooks/useGetLoggedinUser.ts [hunk 1]
```diff
@@ -1,12 +1,24 @@
 "use client";
 import { Cookies } from "@/constant/constant";
 import { getCookies } from "@/utils/cookiesManager";
-import { useMemo } from "react";
+import { useState, useEffect } from "react";
+
 
 export const useGetLoggedInUser = () => {
```
Comment (Monfernape) [inline src/hooks/useGetLoggedinUser.ts:7]: Let's just call it `useUser`

File: src/hooks/useGetLoggedinUser.ts [hunk 2]
```diff
@@ -1,12 +1,24 @@
...
-  const loginUser = useMemo(async () => {
-    const loginUserInfo = await getCookies(Cookies.LoginUser);
-    return loginUserInfo;
+  const [loggedInUser, setLoggedInUser] = useState();
+
+  useEffect(() => {
+    const fetchLoggedInUser = async () => {
+      try {
+        const response = await getCookies(Cookies.LoginUser);
```
Comment (Monfernape) [inline src/hooks/useGetLoggedinUser.ts:13]: Do we know what cookies return?

**Review by Monfernape — APPROVED** [review]

File: src/hooks/useGetLoggedinUser.ts (hunk 2 above)
Comment (muhammad-bilal-ali-khokhar) [reply src/hooks/useGetLoggedinUser.ts:13]: yes sir we know data type i have added the type

File: src/hooks/useGetLoggedinUser.ts (hunk 1 above)
Comment (muhammad-bilal-ali-khokhar) [reply src/hooks/useGetLoggedinUser.ts:7]: updated

**Pull request merged**